core/shared/src/test/scala/zio/prelude/DebugSpec.scala

```diff
@@ -192,6 +192,16 @@ object DebugSpec extends DefaultRunnableSpec {
         testM("testTrait")(check(genTestTrait) { c =>
           assert(c.debug.render(Renderer.Full))(equalTo(s"DebugSpec.${c.getClass.getSimpleName.init}"))
         })
+      ),
+      suite("DebugInterpolator")(
```

Comment:
Should we have a test whether the `Debug` rendering is different from `toString`?

Reply:
`dbg"..."` actually does behave differently from `s"..."` in some cases, because it uses `Renderer.Simple` (and not `Renderer.Scala`).

Reply:
Yes definitely. I'm just wondering we should have a test where the `Debug` rendering is different from the `toString` rendering so we can verify that we actually getting the result of `Debug`.